OutOfSchool/OutOfSchool.Redis/CacheService.cs

```diff
@@ -95,24 +95,71 @@ await ExecuteRedisMethod(() =>
     }
 
     public Task RemoveAsync(string key)
-        => ExecuteRedisMethod(async () => {
+        => this.ExecuteRedisMethod(() =>
```

Comment:
Could you please look inside this method and check if we can stuck in the internal `while` for a wile ?
Also please, invert the `if` inside it,

Reply:
I can't find _if_ in this method